test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -177,6 +176,7 @@ public void Dispose()
             RequestContext.Clear();
         }
 
+#if !NETSTANDARD
```

Comment:
Is testing of flowing of ```Trace.CorrelationManager.ActivityId``` is the only reason of disabling of those tests on NETSTANDARD?